nicoRDL/dotTheory-JS
# JS-105 (JS-349) JavaScript Arrays: Multidimensional Arrays

## Related Tickets

https://codeandtheory.atlassian.net/browse/JS-349

## Description
- finished tutorial
- completed code challenge, sample problems, and quiz

## Steps to Validate
1. Open the sampleProblems/index.html file
2. Load basicMArray.js file and call the coordinates array on the dev console; it should show a multidimensional array

1. Open the codeChallenge/index.html file
2. the quiz prompt should immediately appear
3. finish the quiz; the result of the quiz should appear on screen after taking it

## Risks
N/A
